airbyte-integrations/connectors/source-gitlab/source_gitlab/util.py

```diff
@@ -0,0 +1,21 @@
+import requests
+
+def get_group_list(**kwargs):
+    headers = kwargs["authenticator"].get_auth_header()
```

Comment:
This function is only used by the Source class, maybe transfer it to inside the class or let inside the source.py file. Don't see a separate file is helping here.

Reply:
Moved to an internal class method on the source